pytorch_lightning/plugins/training_type/single_tpu.py

```diff
@@ -54,13 +56,20 @@ def pre_dispatch(self) -> None:
         self.tpu_local_core_rank = xm.get_local_ordinal()
         self.tpu_global_core_rank = xm.get_ordinal()
 
-    def on_save(self, checkpoint: dict) -> dict:
-        """
-        Move XLA tensors to CPU before saving
-        Recommended on XLA Guide:
-        https://github.com/pytorch/xla/blob/master/API_GUIDE.md#saving-and-loading-xla-tensors
+    def save(self, state_dict: Dict, path: str) -> None:
+        xm.save(state_dict, path)
```

Comment:
Is there a reason to put this line under a function call? Do you plan to re-use it?

Reply:
Yup. Planning to explore one more XLA saving functionality.